tools/cli/src/webpack/runtime-config.ts

```diff
@@ -21,12 +21,11 @@ export function getRuntimeConfig(buildFlags: BuildFlags): RuntimeConfig {
         enableExperimentalFeature: true,
         allowLocalWorkspace:
           buildFlags.distribution === 'desktop' ? true : false,
-        enableOrganize: false,
-        enableNewFavorite: false,
-        enableOldFavorite: true,
+        enableOrganize: true,
+        enableNewFavorite: true,
+        enableOldFavorite: false,
 
-        // TODO(@forehalo): enable in next release and canary
-        enableInfoModal: false,
+        enableInfoModal: true,
 
         // CAUTION(@forehalo): product not ready, do not enable it
         enableNewSettingUnstableApi: false,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +61]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->can be removed<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7738?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
will remove in other pr